File: Java/src/main/java/jakopec/model/MjesanjeDvaUnosa.java
package jakopec.model;

import jakarta.persistence.*;

@Entity
@DiscriminatorValue("dvaUnosa")
public class MjesanjeDvaUnosa extends Mjesanje {

    @Column(name = "bodovaprviunos")
    private int bodovaPrviUnos;
    @Column(name = "bodovadrugiunos")
    private int bodovaDrugiUnos;

    @Column(name = "zvanjeprviunos")
    private int zvanjePrviUnos;
    @Column(name = "zvanjedrugiunos")
    private int zvanjeDrugiUnos;


    public int getBodovaPrviUnos() {
        return bodovaPrviUnos;
    }

    public void setBodovaPrviUnos(int bodovaPrviUnos) {
        this.bodovaPrviUnos = bodovaPrviUnos;
    }

    public int getBodovaDrugiUnos() {
        return bodovaDrugiUnos;
    }

    public void setBodovaDrugiUnos(int bodovaDrugiUnos) {
        this.bodovaDrugiUnos = bodovaDrugiUnos;
    }

    public int getZvanjePrviUnos() {
        return zvanjePrviUnos;
    }

    public void setZvanjePrviUnos(int zvanjePrviUnos) {
        this.zvanjePrviUnos = zvanjePrviUnos;
    }

    public int getZvanjeDrugiUnos() {
        return zvanjeDrugiUnos;
    }

    public void setZvanjeDrugiUnos(int zvanjeDrugiUnos) {
        this.zvanjeDrugiUnos = zvanjeDrugiUnos;
    }


    @Override
    public Rezulat getRezultat() {
        return new Rezulat(getBodovaPrviUnos() + getZvanjePrviUnos(),
                getBodovaDrugiUnos() + getZvanjeDrugiUnos());
    }
}

File: CSHARP/CSHARP/Model/MjesanjeDvaUnosa.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace CSHARP.Model
{
    public class MjesanjeDvaUnosa:Mjesanje
    {
        [Column("bodovaprviunos")]
        public int bodovaPrviUnos { get; set; }
        [Column("bodovadrugiunos")]
        public int bodovaDrugiUnos { get; set; }

        [Column("zvanjeprviunos")]
        public int zvanjePrviUnos { get; set; }
        [Column("zvanjedrugiunos")]
        public int zvanjeDrugiUnos { get; set; }

        public override Rezultat getRezultat()
        {
            return new Rezultat
            {
                prvi = bodovaPrviUnos + zvanjePrviUnos,
                drugi = bodovaDrugiUnos + zvanjeDrugiUnos
            };
        }
    }
}
